README.md

```diff
@@ -3,66 +3,105 @@
 ![Pre-Commit](https://github.com/JamesKunstle/explorer/actions/workflows/pre-commit.yml/badge.svg)
 ![Build-Push](https://github.com/JamesKunstle/explorer/actions/workflows/build-push-quay.yml/badge.svg)
 
+Welcome to 8knot!
 
-Welcome to 8Knot!
+[Example of Application](https://eightknot.osci.io/)
+---
 
-*tl;dr sandiego-rh/explorer organizes the visualizations and data generated in the sandiego-rh/sandiego repo as a web-app.*
+## Introduction
 
-[Example of Application](https://eightknot.osci.io/)
+8knot is a [Dash](https://dash.plotly.com/) data web-app built by Red Hat's Open Source Program Office ([OSPO](https://www.redhat.com/en/blog/channel/red-hat-open-source-program-office)).
 
-The number of repositories and projects that GitHub hosts is staggering. Even more impressive is the rate at which this number is growing.
-Projects on GitHub range some student projects that are likely to only be updated a handful of times ever to technological monoliths like the
-Linux Kernel. Understanding the growth trajectory, contributor behavior, progress blockers, etc. intuitively is no longer feasible for individual
```

Comment:
Reason for taking this entirely out?

Reply:
do you like it? I removed it because it was verbose and I didn't like my own writing there but I would keep it in if you like it

Reply:
Personally I liked it. If you want to make some edits to fit better for you I would. I think the additional context is useful

Reply:
Okay I kept it in, that's good feedback. I made a little edit but it's intact.